[2025-01-13T22:15:31.951+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_pipeline.save_to_postgres manual__2025-01-13T22:15:28.891588+00:00 [queued]>
[2025-01-13T22:15:31.958+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_pipeline.save_to_postgres manual__2025-01-13T22:15:28.891588+00:00 [queued]>
[2025-01-13T22:15:31.958+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-01-13T22:15:31.972+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): save_to_postgres> on 2025-01-13 22:15:28.891588+00:00
[2025-01-13T22:15:31.977+0000] {standard_task_runner.py:60} INFO - Started process 111 to run task
[2025-01-13T22:15:31.979+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline', 'save_to_postgres', 'manual__2025-01-13T22:15:28.891588+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_dag.py', '--cfg-path', '/tmp/tmpkkc75m4z']
[2025-01-13T22:15:31.980+0000] {standard_task_runner.py:88} INFO - Job 73: Subtask save_to_postgres
[2025-01-13T22:15:32.018+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_pipeline.save_to_postgres manual__2025-01-13T22:15:28.891588+00:00 [running]> on host 4ba0c249c602
[2025-01-13T22:15:32.074+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_pipeline' AIRFLOW_CTX_TASK_ID='save_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-01-13T22:15:28.891588+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-13T22:15:28.891588+00:00'
[2025-01-13T22:15:32.075+0000] {logging_mixin.py:188} INFO - Saving data to PostgreSQL...
[2025-01-13T22:15:32.081+0000] {base.py:83} INFO - Using connection ID 'my_postgres_conn' for task execution.
[2025-01-13T22:15:32.090+0000] {logging_mixin.py:188} INFO - Data saved to PostgreSQL.
[2025-01-13T22:15:32.090+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-13T22:15:32.099+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_pipeline, task_id=save_to_postgres, execution_date=20250113T221528, start_date=20250113T221531, end_date=20250113T221532
[2025-01-13T22:15:32.152+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-13T22:15:32.166+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
